images/node/example/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "example",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
This looks like it probably passes that string to `sh -c`.  which is really weird. but [doc](https://docs.npmjs.com/cli/v10/using-npm/scripts) mentions "command" and "execute".

Reply:
We could just get rid of this. This enables `npm run test`, which clearly isn't meant to succeed anyway.